camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -347,20 +348,21 @@ const serviceFlowModule = namespace("serviceFlowModule");
 })
 export default class Tasklist extends Mixins(TaskListMixin) {
   @Prop() private getTaskId!: string;
+  @Prop() private mainStore!: any;
```

Comment:
What is the default value for this mainStore prop? Can you update it `Views/Tasklist.vue` file as well

Reply:
thanks for bring this up, we no longer need this